Documentation/core-repos.md

```diff
@@ -59,6 +59,15 @@
 |[dotnet/EntityFramework.Docs](https://github.com/dotnet/EntityFramework.Docs)|EF Core & EF 6 documentation          |
 |[dotnet/sqlclient](https://github.com/dotnet/sqlclient)                      |SqlClient data provider for SQL Server|
 
+## Machine Learning and AI
+
+|Repository                                                                                    |Description              |
+|----------------------------------------------------------------------------------------------|-------------------------|
+|[dotnet/machinelearning](https://github.com/dotnet/machinelearning)                           |ML.NET                   |
+|[dotnet/machinelearning-modelbuilder](https://github.com/dotnet/machinelearning-modelbuilder) |Model Builder            |
```

Comment:
That's what it is, rigth?

```suggestion
|[dotnet/machinelearning-modelbuilder](https://github.com/dotnet/machinelearning-modelbuilder) |ML.NET Model Builder UI for Visual Studio|
```

Reply:
I'll let @briacht comment on the correct description

Reply:
History

Reply:
Shahidzada654@gmail.com